test/run.d

```diff
@@ -410,6 +410,12 @@ string[string] getEnvironment()
     return env;
 }
 
+/// append the given arg with a space if both args are not empty
+string appendSpaced(string left, string right)
+{
+    return left.length ? (right.length ? left ~ " " ~ right : left) : right;
```

Comment:
This seems needlessly complicated. Why not `text (left, " ", right)` ?

Reply:
That works so long as you don't care about the extra space when one argument is empty.